static/home/css/style.css

```diff
@@ -46,10 +46,18 @@ a:hover{
   ::-webkit-scrollbar-thumb:hover {
 	/* background: var(--theme-dark-color-0); */
 	/* background: #666; */
-  }
+  } 
 
 /*  Customized Scroll Bar - Chrome Only -  END */
 
+/* Modifying heading font-size */
+h1 {
+	font-size: 2rem;
+}
+
+h2 {
+	font-size: 1.75rem;
+}
```

Comment:
I think `h1` will look better at 1.75rem here and there's no need to change `h2`, it looks fine. `h1` seems too big with `2rem` in bottom sections.